src/commands/components/publish.ts

```diff
@@ -90,7 +90,7 @@ export default class PublishCommand extends Command {
     const packagePath = await createComponentPackage();
 
     if (checkSignature) {
-      const signatureMatches = await checkPackageSignature(definition, packagePath, customer);
+      const signatureMatches = await checkPackageSignature(definition, packagePath);
```

Comment:
```suggestion
      const signatureMatches = await checkPackageSignature(definition, packagePath, customer);
```

Not sure if this needs to be removed, this is strictly for the  `components:publish` command and not the `components:signature`.

If we remove customer we'll also need to let Taylor know this to update the docs. https://prismatic.io/docs/cli/prism/#componentspublish

Reply:
We need this. Customers can own components.